frappe/commands/site.py

```diff
@@ -195,6 +195,8 @@ def install_app(context, apps):
 			except Exception as err:
 				err_msg = ":\n{}".format(err if str(err) else frappe.get_traceback())
 				print("An error occurred while installing {}{}".format(app, err_msg))
+				print(frappe.get_traceback())
+				frappe.log_error(title=err_msg, message=frappe.get_traceback())
```

Comment:
How about handling this on L196 alone? Have err_msg store both the title and the traceback instead of the `if-else`?

Also, I don't see any reason why this should be logged in the Error Log DocType...This is a bench level operation. It's failures are logged under `logs/bench.log`.

Reply:
> Also, I don't see any reason why this should be logged in the Error Log DocType...This is a bench level operation. It's failures are logged under `logs/bench.log`.

The app is getting installed on a site na, also the error log doctype is much more user friendly

Reply:
> > Also, I don't see any reason why this should be logged in the Error Log DocType...This is a bench level operation. It's failures are logged under `logs/bench.log`.
> 
> The app is getting installed on a site na, also the error log doctype is much more user friendly

I don't think it's user friendly to switch interfaces to see errors. Even if the installation was done through Bench Manager, I don't think Error Logs are anyone-friendly 😆